.devcontainer/docker-compose.yml

```diff
@@ -5,17 +5,17 @@ services:
     build:
       context: .
       dockerfile: Dockerfile
-    ports: 
-      - "5000:5000"
+    ports:
+      - "5555:5000"
```

Comment:
Må ikke denne også mappes til 5555 internt siden alle internportene også er satt til 5555?
Tror det er enklere med en search-and-replace på alt av 5000